libraries/stdlib/common/src/generated/_Strings.kt

```diff
@@ -48,8 +48,8 @@ public inline fun CharSequence.elementAtOrNull(index: Int): Char? {
 
 /**
  * Returns the first character matching the given [predicate], or `null` if no such character was found.
- * 
- * @sample samples.collections.Collections.Elements.find
+ *
+ * @sample samples.text.Strings.find
```

Comment:
@RohitGupta200 please make sure that these generated files are updated via `:tools:kotlin-stdlib-gen:run` command.
See https://github.com/JetBrains/kotlin/blob/master/libraries/stdlib/ReadMe.md#code-generation for details.